pkg/framework/coremodel/registry/registry_gen.go

```diff
@@ -0,0 +1,91 @@
+// This file is autogenerated. DO NOT EDIT.
+//
+// Generated by pkg/framework/coremodel/gen.go
+// Run "make gen-cue" from repository root to regenerate.
+
+package registry
+
+import (
+	"sync"
+
+	"github.com/google/wire"
+
+	"github.com/grafana/grafana/pkg/coremodel/dashboard"
+	"github.com/grafana/grafana/pkg/cuectx"
+	"github.com/grafana/grafana/pkg/framework/coremodel"
+	"github.com/grafana/thema"
+)
+
+// CoremodelSet contains all of the wire-style providers related to coremodels.
+var CoremodelSet = wire.NewSet(
+	ProvideStatic,
+	ProvideGeneric,
+)
```

Comment:
IMO chunking out logical subsets of wire providers into distinct sets like this - then codegenning them! - is a pretty nifty way to cut down on the massive, monotonically increasing pile of stuff in `pkg/server/wire.go`